tests/test_api_versions.py

```
@@ -109,19 +109,17 @@
+        resp = client.get(f"/api/versions/Patient/999999/")
         assert resp.status_code == 200
         data = resp.json()
         assert isinstance(data, list)
         assert data == []
 
     def test_versions_revert_missing_target_version_400(self) -> None:
-        from uuid import uuid4
         client = APIClient()
         resp = client.post(
-            f"/api/versions/Patient/{uuid4()}/revert/", 
+            f"/api/versions/Patient/999999/revert/", 
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)